proto/c1/connector/v2/resource.proto

```
@@ -79,6 +79,10 @@
   rpc DeleteResource(DeleteResourceRequest) returns (DeleteResourceResponse);
 }
 
+service ResourceDeleterService {
+  rpc DeleteResourceV2(DeleteResourceRequestV2) returns (DeleteResourceResponseV2);
```

Buf (1.47.2)

RPC request type "DeleteResourceRequestV2" should be named "DeleteResourceV2Request" or "ResourceDeleterServiceDeleteResourceV2Request".

(RPC_REQUEST_STANDARD_NAME)